src/wechaty-bro.js

```diff
@@ -314,18 +314,25 @@
       var m = Object.assign({}, msg)
       var content = utilFactory.htmlDecode(m.MMActualContent)
       content = utilFactory.encodeEmoji(content)
+      // remove <br/>
+      content = content.replace(/<br.*?\/>/g, '')
       var revokemsg = utilFactory.xml2json(content).revokemsg
       if (revokemsg.msgid) {
-        var chatMsgs = chatFactory.getChatMessage(m.MMPeerUserName)
-        var i = chatFactory._findMessageByMsgId(chatMsgs, revokemsg.msgid)
-        if (i > -1) {
-          m = chatMsgs[i]
-          m.MsgType = confFactory.MSGTYPE_RECALLED
-        } else {
-          m.MsgId = revokemsg.msgid
-          m.MMActualContent = m.Content = revokemsg.replacemsg.replace(/"/g, '')
-        }
-        WechatyBro.emit('message', m)
+        // var chatMsgs = chatFactory.getChatMessage(m.MMPeerUserName)
+        // var i = chatFactory._findMessageByMsgId(chatMsgs, revokemsg.msgid)
+        // if (i > -1) {
+        //   m = chatMsgs[i]
+        //   m.MsgType = confFactory.MSGTYPE_RECALLED
+        // } else {
+        //   m.MsgId = revokemsg.msgid
+        //   m.MMActualContent = m.Content = revokemsg.replacemsg.replace(/"/g, '')
+        // }
+        // WechatyBro.emit('message', m)
+        
+        // add recalled message
+        m.MsgType = confFactory.MSGTYPE_RECALLED
+        m.MMActualContent = JSON.stringify(revokemsg)
```

Comment:
From the design in https://github.com/Chatie/wechaty/pull/1735, the content should be the recalled message id, so looks like here should be:
```ts
m.MMActualContent = revokemsg.msgid
```

Then using the `toRecalled` method on `Message` instance, the recalled message can be retrieved back.

Reply:
Yes, I think so.

Please go to review interface and add your suggestions as a required change for this PR, thanks!